acceptance/tests/service_enable_start.rb

```diff
@@ -0,0 +1,12 @@
+test_name 'Validate that pxp-agent service can be enabled and started'
+
+agents.each do |agent|
+  step 'start service'
+  on(agent, puppet('resource service pxp-agent ensure=running enable=true'))
+
+  step 'validate that service is enabled and running'
+  on(agent, puppet('resource service pxp-agent')) do |result|
```

Comment:
(shrug) when I tested line 5 on Soi 11, puppet returned an exit code of 0 even though the service state returned 'ensure=stopped' and 'enable=false'. So I put in the addition check to get back the actual state after the 60 second time out.